css/style.css

```diff
@@ -2018,3 +2018,84 @@ the home page, intro pitch and your-code-here layed out vertically */
 dt.d_decl:hover .decl_anchor {
     visibility: visible;
 }
+
+/* Friends of D */
+.orgs-using-d {
+    width: 100%;
+    padding-left: 20px;
```

Comment:
I think this padding should not be here. It's only on one side, and it gets added to the 100% width, resulting in an element that's more than 100% wide.